### DISCUSSION TITLE
Selecting checkbox while pressing the command/ctrl key

### DISCUSSION THREAD

**Discussion by shivamdz** [UI5/webcomponents-react#3402] [Q&A] [answered]
Hi Everyone,

I want to select a checkbox while pressing the command/ctrl key. In my onChange event of the checkbox, I should know if the command/ctrl key was pressed or not. I tried reading the "window.event" variable but couldn't find any information related to the pressed command/ctrl key.

Any Suggestions?

Thanks,
Shivam

**Comment by Lukas742** [comment, marked as answer]
Hi @shivamdz 

the `change` event is a `CustomEvent` that doesn't hold the `ctrlKey` or `metaKey` information as it only inherits the properties of the `Event` type. This is similar to the native `change` event as it is as well only an `Event`.

So, to check whether CTRL or COMMAND is pressed (probably depending on OS), you need to register a global listener for it, then you can just update a React state depending on if one of the keys is pressed or not and check for that in your `onChange` callback.

https://codesandbox.io/s/command-ctrl-um916y?file=/src/App.js
